<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublishReply</class>
 <widget class="QWidget" name="PublishReply">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="headBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background:#ed6a5a;</string>
   </property>
   <property name="text">
    <string>Reply</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/comment_icon.png</normaloff>:/res/comment_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="footnav" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>411</width>
     <height>89</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ed6a5a;</string>
   </property>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>111</width>
      <height>89</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;
background:#ed6a5a;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/cancel_icon.png</normaloff>:/res/cancel_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="restitlewidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>411</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>411</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>411</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background:#9bc1bc;
</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="headtitlelabel">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Title: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="headusernamelabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>@username</string>
       </property>
      </widget>
      <widget class="QLabel" name="headtimelabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>release time: 2022-12-21 24:24:24</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="headviewedlabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>10 viewed</string>
       </property>
      </widget>
      <widget class="QLabel" name="headreplylabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>10 reply</string>
       </property>
      </widget>
      <widget class="QLabel" name="headlikelabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>10 like</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="headtitleedit">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid;
border-radius: 5px;
background-color: rgb(230, 235, 224);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>411</width>
     <height>348</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 235, 224);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Content:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>391</width>
      <height>290</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-radius: 5px groove gray;
background-color: rgb(230, 235, 224);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="btnwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>528</y>
     <width>411</width>
     <height>42</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>411</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>411</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(155, 193, 188);
border: 0px;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,40,40">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>134</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="submitBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background-color: rgb(155, 193, 188);
border: 1px solid;</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/submit_icon.png</normaloff>:/res/submit_icon.png</iconset>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="clearBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background-color: rgb(155, 193, 188);
border: 1px solid;</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/delete_icon.png</normaloff>:/res/delete_icon.png</iconset>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
